CRM/Utils/Money.php

```diff
@@ -347,7 +347,9 @@ protected static function formatNumericByFormat($amount, $valueFormat = '%!i') {
     if (is_numeric($amount) && function_exists('money_format')) {
       $lc = setlocale(LC_MONETARY, 0);
       setlocale(LC_MONETARY, 'en_US.utf8', 'en_US', 'en_US.utf8', 'en_US', 'C');
+      // phpcs:disable
       $amount = money_format($valueFormat, $amount);
+      // phpcs:enable
```

Comment:
This was annoying - checkstyle fails on "deprecated function". It's unrelated to this PR but I had to add this to get tests to pass.

Reply:
This seems ok to add. This only gets called in 2 edge-cases - no `intl` or somebody is trying to monetize html. There shouldn't be any of the latter anymore, and we eventually have to decide what to do for people who can't install `intl`. Then this can go.